src/ChipInput.js

```diff
@@ -183,6 +183,12 @@ class ChipInput extends React.Component {
     }
   }
 
+  componentDidUpdate (previousProps) {
+    if(!previousProps.defaultValue && this.props.defaultValue) {
+      this.setState({ chips: this.props.defaultValue });
```

Comment:
This would overwrite the chips if `defaultValue` is changed after the user entered some chips.